### PR TITLE
Fix video system selection for MSP

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: https://github.com/betaflight/betaflight/issues/11822

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
afraid not
```
# version
# Betaflight / STM32F405 (S405) 4.3.1 Jul 13 2022 / 03:32:11 (8d4f005) MSP API: 1.44
# config: manufacturer_id: FOXE, board_name: FOXEERF405, version: 9d0a4b63, date: 2022-10-12T09:07:03Z
# board: manufacturer_id: FOXE, board_name: FOXEERF405

# get vcd_video_system
vcd_video_system = AUTO
Allowed values: AUTO, PAL, NTSC

# get osd_displayport_device
osd_displayport_device = AUTO
Allowed values: NONE, AUTO, MAX7456, MSP, FRSKYOSD
```

**Comment by nerdCopter** [comment]
deleted last comment, i dis not use correct firmware

**Review by nerdCopter — APPROVED** [review]

**Review by sugaarK — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
